docs/source/profiler.rst

```diff
@@ -761,6 +761,13 @@ Below is an breakdown of all the options.
       * top_k_categories - (int) Number of categories to be displayed when reporting
       * max_sample_size_to_check_stop_condition - (int) The maximum sample size before categorical stop conditions are checked
       * stop_condition_unique_value_ratio - (float) The highest ratio of unique values to dataset size that is to be considered a categorical type
+      * cms - (Boolean) Enables or Disables the use of count min sketch / heavy hitters for approximate frequency counts.
+      * cms_confidence (float) Defines the number of hashes used in CMS.
+        eg. confidence = 1 - failure probability, default 0.95
+      * cms_relative_error: (float) Defines the number of buckets used in CMS,
+            default 0.01
+      * cms_max_num_heavy_hitters (int) The value used to define
+        the threshold for minimum frequency required by a category to be counted
```

Comment:
indent these for clarity where they fall?

Reply:
maybe don't indent if its all attributes under `CategoricalOptions`

Reply:
following pattern of pre-existing options at the same level